terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 1: 28538 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-8-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 2: 28540 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-8-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 3: 28542 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-8-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 4: 28544 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-8-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 5: 28546 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-8-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 6: 28548 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-9-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 7: 28550 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-9-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 8: 28552 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-9-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 9: 28554 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-9-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 10: 28556 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-9-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 11: 28558 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-10-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 12: 28560 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-10-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 13: 28562 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-10-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 14: 28564 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-10-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 15: 28566 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-10-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 16: 28568 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-11-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 17: 28570 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-11-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 18: 28572 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-11-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 19: 28574 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-11-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 20: 28576 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-11-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 21: 28578 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-12-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 22: 28580 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-12-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 23: 28582 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-12-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 24: 28584 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-12-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 25: 28586 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-12-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 26: 28588 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-13-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 27: 28590 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-13-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 28: 28592 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-13-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 29: 28594 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-13-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 30: 28596 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-13-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 31: 28598 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-14-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 32: 28600 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-14-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 33: 28602 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-14-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 34: 28604 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-14-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 35: 28606 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-14-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 36: 28608 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-15-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 37: 28610 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-15-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 38: 28612 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-15-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 39: 28614 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-15-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 40: 28616 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-15-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 41: 28618 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-16-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 42: 28620 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-16-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 43: 28622 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-16-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 44: 28624 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-16-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 45: 28626 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-16-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 46: 28628 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-17-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 47: 28630 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-17-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 48: 28632 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-17-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 49: 28634 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-17-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 50: 28636 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-17-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 51: 28638 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-18-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 52: 28640 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-18-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 53: 28642 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-18-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 54: 28645 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-18-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 55: 28647 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-18-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 56: 28650 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-19-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 57: 28652 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-19-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 58: 28654 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-19-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 59: 28656 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-19-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 60: 28658 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-19-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 61: 28662 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-20-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 62: 28668 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-20-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 63: 28670 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-20-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 64: 28672 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-20-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 65: 28674 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-20-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 66: 28676 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-21-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 67: 28682 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-21-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 68: 28686 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-21-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 69: 28696 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-21-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 70: 28698 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-21-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 71: 28700 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-3-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 72: 28702 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-5-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 73: 28705 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-7-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 74: 28707 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-4-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 75: 28709 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-2-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 76: 28711 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-2-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 77: 28713 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-6-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 78: 28715 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-5-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 79: 28718 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-5-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 80: 28720 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-1-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 81: 28722 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-6-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 82: 28724 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-7-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 83: 28726 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-7-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 84: 28728 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-3-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 85: 28730 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-6-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 86: 28732 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-3-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 87: 28735 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-7-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 88: 28737 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-4-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 89: 28739 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-4-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 90: 28741 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-5-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 91: 28743 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-2-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 92: 28745 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-1-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 93: 28747 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-1-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 94: 28749 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-7-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 95: 28751 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-1-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 96: 28753 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-3-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 97: 28755 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-2-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 98: 28757 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-6-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 99: 28759 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-4-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 100: 28761 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-2-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 101: 28763 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-4-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 102: 28765 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-5-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/13.pos.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-2-0-b-val.sh: line 103: 28767 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-3-neg-b.pddl $1
